src/main/rx/rx.c

```diff
@@ -1020,23 +1022,22 @@ bool isRssiConfigured(void)
     return rssiSource != RSSI_SOURCE_NONE;
 }
 
-timeDelta_t rxGetFrameDelta(timeDelta_t *frameAgeUs)
+timeDelta_t rxGetFrameDelta(void)
 {
-    static timeUs_t previousFrameTimeUs = 0;
+    // return the delta time based on time stamps provided by the Rx driver
+    // frameTimeDeltaUs holds its last good value until after 150ms of no frames, it goes to zero)
```

Comment:
Should I revert to exactly the old code?

Reply:
>Old code did reuse frameTimeDeltaUs on subsequent calls until new rx frame arrived ...

The problem here is that if the previous delta time was unusual, it would then be held for the duration of some subsequent delta time, which could be a long time.  We would hold a previous value that may, or may not, have any relevance to the current interval.  And, being non-zero, we couldn't detect this easily.

By forcing to zero if there is no valid RC delta, we can know that there hasn't been a new valid packet; and if we receive a NULL value, we won't use the delta time associated with that NULL.

Reply:
Changing the function signature so that it is obvious when new RC is available and when timeout/recheck triggered it may help a lot ...